rules/Composer/Application/FileProcessor/ComposerFileProcessor.php

```diff
@@ -57,7 +57,7 @@ public function supports(File $file): bool
             return true;
         }
 
-        return $fileInfo->getRealPath() === getcwd() . '/composer.json';
+        return $fileInfo->getFilename() === 'composer.json';
```

Comment:
this is not working for me, because I usually (always) run rector with path "src" or "app" but composer is one level up, so it is not loaded for processing

Reply:
What would work then?

Reply:
1) add composer.json file to path list hardcoded (as it was before): getcwd() . '/composer.json';
2) let it on user to run rector with two paths, in my case
bin/rector process /var/www/league-event-app/app/ /var/www/league-event-app/composer.json

Reply:
3. add option "exclude-paths" and let user to run rector on app root:
```
bin/rector process /var/www/league-event-app/ --exclude-paths="vendor"
```

Reply:
Ah, I though it's a `getBasename()` :D that would do?

Reply:
not at this place :)

composer.json will never go to this method if rector is executed on app / src dir

Reply:
That's a different place :)

This method only checks passing value, not loading it.

Reply:
that's what I am talking about :)